src/app/clusters/scenes-server/SceneHandlerImpl.h

```diff
@@ -0,0 +1,228 @@
+/**
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/clusters/scenes-server/SceneTable.h>
+#include <app/util/attribute-storage.h>
+#include <lib/core/DataModelTypes.h>
+
+namespace chip {
+namespace scenes {
+
+using ClusterId = chip::ClusterId;
```

Comment:
Why bother?  If you're inside here, `ClusterId` would be `chip::ClusterId` anyway....